XLS-0066d-lending-protocol/README.md

```diff
@@ -0,0 +1,1560 @@
+- The submitter `AccountRoot.Account != LoanBroker(LoanBrokerID).Owner`.
+
+- If the `Vault(LoanBroker(LoanBrokerID).VaultID).Asset` is `XRP`:
+
+  - `AccountRoot(LoanBroker.Owner).Balance < Amount` (LoanBroker does not have sufficient funds to deposit the First Loss Capital).
+
+- If the `Vault(LoanBroker(LoanBrokerID).VaultID).Asset` is an `IOU`:
+
+  - The `RippleState` object between the submitter account and the `Issuer` of the asset has the `lsfLowFreeze` or `lsfHighFreeze` flag set.
+  - The `AccountRoot` object of the `Issuer` has the `lsfGlobalFreeze` flag set.
+  - The trustline `Balance` < `Amount`.
+
+- If the `Vault(LoanBroker(LoanBrokerID).VaultID).Asset` is an `MPT`:
+
+  - The `MPToken` object for the `Vault(LoanBroker(LoanBrokerID).VaultID).Asset` of the submitter `AccountRoot`:
+    - Has `lsfMPTLocked` flag set.
+    - `MPTAmount` < `Amount`.
+  - The `MPTokenIssuance` object of the `Vault(LoanBroker(LoanBrokerID).VaultID).Asset` has the `lsfMPTLocked` flag set.
+
+##### 3.1.3.2 State Changes
+
+- If the `Vault(LoanBroker(LoanBrokerID).VaultID).Asset` is `XRP`:
+
+  - Increase the `Balance` field of `LoanBroker` _pseudo-account_ `AccountRoot` by `Amount`.
+  - Decrease the `Balance` field of the submitter `AccountRoot` by `Amount`.
+
+- If the `Vault(LoanBroker(LoanBrokerID).VaultID).Asset` is an `IOU`:
+
+  - Increase the `RippleState` balance between the `LoanBroker` _pseudo-account_ `AccountRoot` and the `Issuer` `AccountRoot` by `Amount`.
+  - Decrease the `RippleState` balance between the submitter `AccountRoot` and the `Issuer` `AccountRoot` by `Amount`.
+
+- If the `Vault(LoanBroker(LoanBrokerID).VaultID).Asset` is an `MPT`:
+
+  - Increase the `MPToken.MPTAmount` by `Amount` of the `LoanBroker` _pseudo-account_ `MPToken` object for the `Vault.Asset`.
+  - Decrease the `MPToken.MPTAmount` by `Amount` of the submitter `MPToken` object for the `Vault.Asset`.
+
+- Increase `LoanBroker.CoverAvailable` by `Amount`.
+
+##### 3.1.3.3 Invariants
+
+**TBD**
+
+[**Return to Index**](#index)
+
+#### 3.1.4 `LoanBrokerCoverWithdraw`
+
+The `LoanBrokerCoverWithdraw` transaction withdraws the First-Loss Capital from the `LoanBroker`.
+
+| Field Name        |     Required?      | JSON Type | Internal Type | Default Value | Description                                                   |
+| ----------------- | :----------------: | :-------: | :-----------: | :-----------: | :------------------------------------------------------------ |
+| `TransactionType` | :heavy_check_mark: | `string`  |   `UINT16`    |   **TODO**    | Transaction type.                                             |
+| `LoanBrokerID`    | :heavy_check_mark: | `string`  |   `HASH256`   |     `N/A`     | The Loan Broker ID from which to withdraw First-Loss Capital. |
+| `Amount`          | :heavy_check_mark: | `object`  |   `NUMBER`    |       0       | The amount of Vault asset to withdraw.                        |
```

Comment:
Also here: `sfAmount` is an `AMOUNT` field, which requires currency, issuer, etc. I assume that's not what you want, in which case, we've got `sfNumber`

Reply:
Perhaps it's worth keeping it as amount, and chaking the type to `amount` too? Just in case, in the future, we want to support depositing different cover assets?

Reply:
> Perhaps it's worth keeping it as amount, and chaking the type to `amount` too? Just in case, in the future, we want to support depositing different cover assets?

But doesn't the deposit go into the Single Asset Vault? That can't hold different types of assets.

Reply:
@ximinez , let's keep it as `Amount`?

Reply:
@ximinez , what do you think?

Reply:
> @ximinez , what do you think?

Yeah, I'm fine with keeping it `Amount`, but the type MUST be changed to `AMOUNT`.

Reply:
Done!